stripe/src/test/java/com/stripe/android/paymentsheet/ui/BillingAddressViewTest.kt

```diff
@@ -243,6 +271,24 @@ class BillingAddressViewTest {
             )
     }
 
+    private fun setupPostalCode(
```

Comment:
I like the addition of this function!

Reply:
hmm actually tried to do that in PaymentSheetAddFragmentTest, but there seems to be too many parameters to toggle if we want to cover all tests there, I think it would be more readable to just keep them as is